sqlengine/postgres_engine.go

```diff
@@ -1,43 +1,55 @@
 package sqlengine
 
 import (
+	"bytes"
 	"database/sql"
 	"fmt"
+	"text/template"
 
 	"github.com/lib/pq" // PostgreSQL Driver
 
 	"code.cloudfoundry.org/lager"
 )
 
-const defaultStateDBName = "broker_state"
+var ensureTriggerPattern = `
```

Comment:
I'm not sure why these template patterns were made as globals. This may just be my opinion, but I found this really interrupted the readability of the package code. Also I can't see a reason why they need to be global, since they are not referenced anywhere else in the repository.